hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/devtools/sidecar/DtSidecarWindow.kt

```diff
@@ -49,34 +56,48 @@ import org.jetbrains.compose.reload.core.WindowId
 import org.jetbrains.compose.reload.core.createLogger
 import org.jetbrains.compose.reload.orchestration.OrchestrationMessage.ApplicationWindowGainedFocus
 import org.jetbrains.compose.reload.orchestration.OrchestrationMessage.ShutdownRequest
+import java.awt.Dimension
+import java.awt.Point
 import kotlin.system.exitProcess
+import kotlin.time.Duration.Companion.milliseconds
+
 
 private val logger = createLogger()
 
 // Modern rounded corners like JetBrains Toolbox
 private val DevToolingSidecarShape = RoundedCornerShape(8.dp)
 
+// animation time of window effects
+private val animationDuration = 512.milliseconds
+
 @Composable
 fun DtSidecarWindow(
     windowId: WindowId,
     windowState: WindowState,
     isAlwaysOnTop: Boolean,
 ) {
-
     var isExpanded by remember { mutableStateOf(false) }
+    val initialized = remember { mutableStateOf(true) }
```

Comment:
I noticed that I was confused about this variable: 
Shouldn't this be called 'isInitializing'? 

Tip: maybe using `by remember` would look nicer?